code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -205,11 +211,11 @@
 					return
 				else
 					cleaning = 1
-					drain(500)
+					drain(100)
```

Comment:
Changing drain on every dogborg belly is shenanigans. Isn't this PR just about DA bellies?